### ISSUE TITLE
WebOS experienced an address collision

### ISSUE THREAD

**Issue by badgertastic** [grzegorz914/homebridge-lgwebos-tv#195]
2.16.6 - 2.16.14

```
Error: Accessory Bedroom WebOS experienced an address collision.
    at BridgeService.handlePublishExternalAccessories (/usr/local/lib/node_modules/homebridge/src/bridgeService.ts:419:15)
    at processTicksAndRejections (node:internal/process/task_queues:95:5)
```

EDIT: then plugin crashes and restarts. looping crash

**Comment by siobhanellis** [comment]
I have the same issue. No issue with version 2.15

**Comment by grzegorz914** [comment]
can You both try 2.16.15-beta.0 and let me know

**Comment by siobhanellis** [comment]
Seems to work for me, and I can add the bridge. I don’t see the accessory though ☹

Siobhan

IOT: Practical HomeKit: ( https://practicalhomekit.blogspot.com )
Charity Fundraising: Lammie Drive: ( lammiedrive.blogspot.com<http://lammiedrive.blogspot.com/> )


From: Grzegorz ***@***.***>
Date: Wednesday, 10 January 2024 at 10:40
To: grzegorz914/homebridge-lgwebos-tv ***@***.***>
Cc: siobhanellis ***@***.***>, Comment ***@***.***>
Subject: Re: [grzegorz914/homebridge-lgwebos-tv] WebOS experienced an address collision (Issue #195)

can You both try 2.16.15-beta.0 and let me know

**Comment by grzegorz914** [comment]
Fixed in 2.16.15

**Comment by badgertastic** [comment]
> Fixed in 2.16.15

Thanks. Looks ok.
